src/components/Countries/Countries.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Container from 'react-bootstrap/Container';
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+import { getCountries } from '../../redux/countries/countries';
+import Country from '../Country/Country';
+import images from '../../assets/images.jpg';
+import './countries.css';
+
+const Countries = () => {
```

Comment:
- [ ] Reading from Project requirement: "You should filter the data that you retrieve from the API using a Filter stateless component". **Please you need to use a filter method to filter data retrieved from the API.**